cr-examples/onnx/src/main/java/oracle/code/onnx/OnnxProtoBuilder.java

```diff
@@ -316,7 +317,7 @@ static ByteBuffer buildModel(List<Input> inputs, List<OpNode> ops, List<String>
                 .opset_import(new OperatorSetIdProto().version(OPSET_VERSION))
                 .buf.toByteArray();
```

Comment:
is `buf` a protobuf `ByteString` ? If so, note that ByteString supports `copyTo` a byte buffer. So a possibility would be to create the memory segment, then get a BB view of the segment, and then pass that view to the `copyTo` method. This should avoid the intermediate array creation (but cannot avoid the memory zeroing).

Reply:
nvm - this seems to be an output stream